src/GovUk.Education.ExploreEducationStatistics.Data.Model/LocalAuthority.cs

```diff
@@ -1,9 +1,8 @@
 #nullable enable
-using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Model
 {
-    public class LocalAuthority : ObservationalUnit, ILocationAttribute
+    public class LocalAuthority : LocationAttribute, ILocationAttribute
```

Comment:
Don't think we need the ILocationAttribute if this is already extending from LocationAtribute?

Reply:
`ILocationAttribute` has two default interface methods `GetCacheKey` and `GetCodeOrFallback` which we want to override here in `LocalAuthority` to change their default behaviour.

I can't see anything about this in the docs for default interface methods but unless we repeat the interface declaration here, i.e. `LocalAuthority : LocationAttribute, ILocationAttribute` as opposed to `LocalAuthority : LocationAttribute` the methods are not treated as overrides.

If we remove it then unit tests that we've written around covering the old code for Local Authority data will fail

E.g.
`ImporterLocationServiceTests.FindOrCreate_UniqueByLocalAuthorityOldCode`
`ResultSubjectMetaServiceTests.GetSubjectMeta_LocationsForSpecialCases`
`SubjectMetaServiceTests.GetSubjectMeta_LocationsForSpecialCases`

Would be interested to know if you can spot something wrong with the way those methods are overriden?

Thanks.

Reply:
Will check later - lets get this onto Dev!